fvm/env.go

```diff
@@ -2,12 +2,15 @@ package fvm
 
 import (
 	"github.com/onflow/cadence/runtime"
+
+	"github.com/onflow/flow-go/fvm/state"
 )
 
 // Environment accepts a context and a virtual machine instance and provides
 // cadence runtime interface methods to the runtime.
 type Environment interface {
 	Context() *Context
 	VM() *VirtualMachine
+	StateHolder() *state.StateHolder
```

Comment:
StateHolder and state are internal concepts, not sure exposing the environment is that meaningful here, if we endup needing it somewhere, probably some structures need to change.

Reply:
Removed with [8cec310](https://github.com/onflow/flow-go/pull/1631/commits/8cec310c368c0e444efb933f2b8e9b576c8512d4).